WebSockies/Data/LobbySettings.cs

```diff
@@ -15,6 +15,14 @@ public LobbySettings()
         
         public List<Question> Questions;
         public int TimePerQuestion;
+        public LobbyTypeEnum LobbyType;
+
+
+        public enum LobbyTypeEnum
+        {
+            Standard = 0,
+            Traditional = 1
```

Comment:
Die 0 en 1 zijn overbodig, dat zijn al de standaard waardes, enums vergelijken met ints is ook tegen het doel van enums in